bin/index.js

```diff
@@ -213,8 +229,18 @@ module.exports = function(Aquifer, AquiferGitConfig) {
 
       // Commit changes.
       .then(function () {
+        var signature;
+
         Aquifer.console.log('Commit changes...', 'status');
-        var signature = git.Signature['default'](repo);
+
+        if (options.name && options.email) {
```

Comment:
@chasingmaxwell If the property on `options` is always going to exist but could be set to `null`, I think you can do something like, 

```javascript
if (!!options.name && !!options.email){
// both are not null
}
```